packages/zowe-explorer/CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 
 ### Bug fixes
 
+- Fixed an issue where Zowe Explorer displayed a "No Zowe client configurations" prompt when a project, user-level config exists but no global config was present. [#3168](https://github.com/zowe/zowe-explorer-vscode/issues/3168)
```

Comment:
what do we mean by "user-level"? If we mean a user config file, can we reword this to something like "... prompt when a project user configuration file existed but no global configuration was present."

also, please note the verb tenses above.

Reply:
Thanks, I adjusted the verb tenses and updated the entry 👍